Project.toml

```diff
@@ -3,7 +3,6 @@ keywords = ["package", "management"]
 license = "MIT"
 name = "Pkg"
 version = "1.1.0"
-uuid = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
Also, the UUID should not be removed. You have to remove it to load package when developing, but you should not commit this change.